draft-ietf-tls-tls13.md

```diff
@@ -5157,15 +5294,19 @@ server is not able to guarantee full uniqueness of the handshake
 state. See {{early-data-indication}} for one mechanism to limit
 the exposure to replay.
 
+### Post-Compromise Security
+
 TLS does not provide security for handshakes which take place after the peer's
 long-term secret (signature key or external PSK) is compromised. It therefore
 does not provide post-compromise security {{CCG16}}, sometimes also referred to
 as backwards or future security. This is in contrast to KCI resistance, which
 describes the security guarantees that a party has after its own long-term
 secret has been compromised.
 
+### External References
+
 The reader should refer to the following references for analysis of the
-TLS handshake {{CHSV16}} {{FGSW16}} {{LXZFH16}}.
+TLS handshake {{CHSV16}} {{CHSV16-2}} {DFGS16}} {{DFGS16}} {{Kraw16}} {{KW16}} {{FGSW16}} {{LXZFH16}}.
```

Comment:
To match the styling below, you need a colon after "handshake" and a comma after each item